kensu/utils/helpers.py

```diff
@@ -87,6 +87,38 @@ def report_all2all_lineage(in_obj, out_obj, op_type, in_inmem=True, out_inmem=Tr
                                                from_guid=in_schema.to_guid(),
                                                from_col=str(col),
                                                type=op_type)
+
+
+def report_simple_copy_with_guessed_schema(input_uri,  # type: str
+                                           output_absolute_uri,  # type: str
+                                           read_schema_from_filename,  # type: str
+                                           operation_type,  # type: str
+                                           ):
+    from kensu.utils.dsl.extractors.external_lineage_dtos import GenericComputedInMemDs
```

Comment:
this is same as it was in https://github.com/kensuio-oss/kensu-py/pull/102/files#diff-b1a693175196936a4497753c9d6f4987374273e7c126d77f8c6a3e6172005b17R56-L77 , just extracted